DiffPriv/mech/validate.py

```diff
@@ -0,0 +1,152 @@
+"""Validates mechanism params"""
+
+class DomainError(Exception):
```

Comment:
These errors should be in `DiffPriv/utils.py`.

Reply:
Doing this first (1)

Reply:
Committing now ...